staging/src/k8s.io/apiserver/plugin/pkg/audit/log/backend.go

```diff
@@ -45,6 +49,11 @@ func NewBackend(out io.Writer) *backend {
 
 func (b *backend) ProcessEvents(events ...*auditinternal.Event) {
 	for _, ev := range events {
+		// skip initial event for read-only requests
```

Comment:
I wanted to have a simple and consistent protocol between filter and backend. That's why I did this logic here, to avoid spamming the log.